spec/controllers/resources_controller_spec.rb

```diff
@@ -81,13 +81,19 @@
         context 'and update succeeds' do
           let(:params) do
             {
-              state: 'NY',
-              time: 'Months. Note: Depending on the case, it could take longer.',
-              cost: 'It is free of charge.',
-              award: 'You can potentially claim full amount of reasonable losses in awards for shattered computer, ER visits, or lost days from work.',
-              likelihood: 'The likelihood to get reimbursement through this option depends on whether a criminal case is brought. The system takes care of everything but that only happens if the evidence is strong enough for a prosecutor to bring charges.',
-              safety: 'It is likely that the prosecutor will call you to testify in the criminal case with your abuser present.',
-              story: 'You will have to share your story when you make a report to law enforcement and if you are called to testify, you will have to do so on the stand at trial.',
+              state: 'WV',
+              who: 'You are eligible to apply if you were injured as the result of a crime.You are also eligible to apply if you are the legal dependent of someonewho was killed as the result of a crime',
```

Comment:
Needs a space between `crime.` and `You`. Later in the line someonewho should be broken up into someone who.

Reply:
Thanks for these comments. I'm not sure why the spacing got so messed up.